src/app/rosetta/indexer_test/indexer_test.ml

```diff
@@ -707,9 +707,12 @@ module Max_block =
 module Offset_limit = struct
   type t = { offset : int; limit : int }
 
-  let offset_generator = Int.gen_incl 1 100
+  (* Max value + Max limit value should not exceed valid user_commands count in database
+     otherwise quickcheck will become flaky.
+  *)
+  let offset_generator = Int.gen_incl 1 50
 
-  let limit_generator = Int.gen_incl 1 50
+  let limit_generator = Int.gen_incl 1 20
```

Comment:
I assume we reduce the upper bound here because we actually reached situation where the sum of the 2 exceeded the number of valid user commands. And we bump the limit down to be conservative. 

 Is it possible for us to know the number of valid user_commands, and then generate the 2 params accordingly, to gain more test coverages?